config/routes.rb

```diff
@@ -1,68 +1,20 @@
 Rails.application.routes.draw do
   devise_for :users
-  root "posts#index"
-  get 'about' => 'posts#about'
-  get 'contacts' => 'posts#contacts'
-  get 'press' => 'posts#press'
-  get 'terms' => 'posts#terms'
+  root "static_pages#index"
+  get 'about' => 'static_pages#about'
+  get 'contacts' => 'static_pages#contacts'
+  get 'press' => 'static_pages#press'
+  get 'terms' => 'static_pages#terms'
 
   resources :categories, only: [:new, :create, :index,:show] do
     resources :sections, only: [:new, :create]
   end
 
-  resources :users, only: :show
-  # The priority is based upon order of creation: first created -> highest priority.
-  # See how all your routes lay out with "rake routes".
-
-  # You can have the root of your site routed with "root"
-  # root 'welcome#index'
-
-  # Example of regular route:
-  #   get 'products/:id' => 'catalog#view'
-
-  # Example of named route that can be invoked with purchase_url(id: product.id)
-  #   get 'products/:id/purchase' => 'catalog#purchase', as: :purchase
-
-  # Example resource route (maps HTTP verbs to controller actions automatically):
-  #   resources :products
-
-  # Example resource route with options:
-  #   resources :products do
-  #     member do
-  #       get 'short'
-  #       post 'toggle'
-  #     end
-  #
-  #     collection do
-  #       get 'sold'
-  #     end
-  #   end
-
-  # Example resource route with sub-resources:
-  #   resources :products do
-  #     resources :comments, :sales
-  #     resource :seller
-  #   end
-
-  # Example resource route with more complex sub-resources:
-  #   resources :products do
-  #     resources :comments
-  #     resources :sales do
-  #       get 'recent', on: :collection
-  #     end
-  #   end
+  resources :sections, only: [] do
+    resources :posts, only: [:new, :create, :show]
+  end
 
```

Comment:
And I would add this line here:

```
resources :posts, only: [:show]
```

To hook up `/posts/:id` without passing through the `section_id`.